cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -0,0 +1,137 @@
+                             and in the LinkedIn share post.
+                             Default is "Look what I found on the CFPB!"
+
+   options.twitter_hashtags: A comma-separated list of hashtags
+                             to be appended to default Tweet text.
+                             https://dev.twitter.com/web/tweet-button/parameters
+
+   options.twitter_related: A comma-separated list of accounts
+                            related to the content of the shared URI.
+                            https://dev.twitter.com/web/tweet-button/parameters
+
+   options.twitter_lang:    Loads text components in the specified language.
+                            https://dev.twitter.com/web/tweet-button/parameters
+
+   ========================================================================== #}
+
+{% macro render(is_share_view=true, options={}) %}
+{% set title  =  options.title | default('Look what I found on the CFPB!') %}
+<div class="m-social-media">
+    {% if is_share_view %}
+        <h5 class="m-social-media_heading">Share this</h5>
+    {% endif %}
+
+    <ul class="list m-social-media_icons">
+        {% if is_share_view %}
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://www.facebook.com/dialog/share?app_id=210516218981921&amp;display=page&amp;href={{ request.url | urlencode }}&amp;redirect_uri={{ request.url | urlencode }}"
+                   target="_blank">
+                    <span class="cf-icon cf-icon-facebook-square"></span>
+                    <span class="u-visually-hidden">Share on Facebook</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="{{ _share_twitter_url(options) }}"
+                   target="_blank">
+                    <span class="cf-icon cf-icon-twitter-square"></span>
+                    <span class="u-visually-hidden">Share on Twitter</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="https://www.linkedin.com/shareArticle?mini=true&url={{ request.url | urlencode }}&title={{ title }}%20Developer%20Network"
+                   target="_blank">
+                    <span class="cf-icon cf-icon-linkedin-square"></span>
+                    <span class="u-visually-hidden">Share on LinkedIn</span>
+                </a>
+            </li>
+            <li class="list_item">
+                <a class="m-social-media_icon"
+                   href="mailto:?subject={{ title }}&amp;body=Check out this page from the CFPB - {{ request.url | urlencode }}">
+                    <span class="cf-icon cf-icon-email-social-square"></span>
+                    <span class="u-visually-hidden">Share by email</span>
+                </a>
+            </li>
+        {% else %}
```

Comment:
This is a lot of duplication. Could we populate an object at the top of the block and use those values in a single set of markup?